packages/gateway-api/v1.1.0/package.yaml

```diff
@@ -0,0 +1,10 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "gateway-api"
+shortDescription: The Gateway API provides a standardized way to configure and manage network traffic routing in Kubernetes clusters.
+longDescription: >-
+  The Gateway API is a Kubernetes extension that standardizes the configuration and management of network traffic routing. It introduces new resource types like Gateway, GatewayClass, and HTTPRoute to provide flexible and extensible traffic control. This API enables better separation of concerns and more robust traffic management in Kubernetes environments.
+iconUrl: "https://avatars.githubusercontent.com/u/36015203"
+defaultNamespace: "gateway-api-system"
```

Comment:
Not sure about this, it seems to me that this package only installs CRDs right? In that case, I think the `defaultNamespace` can be removed. I also tried installing the package, the namespace is not being created.